test/e2e/upgrade/manifestdelete/manifest-delete.go

```diff
@@ -0,0 +1,187 @@
+package manifestdelete
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	restclient "k8s.io/client-go/rest"
+	apiregistrationsclientset "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/upgrades"
+)
+
+// UpgradeTest runs verifies invariants regarding what alerts are allowed to fire
+// during the upgrade process.
+type UpgradeTest struct {
+	oc     *exutil.CLI
+	config *restclient.Config
+}
+
+type object struct {
+	kind      string
+	namespace string
+	name      string
+}
+
+type unstructured struct {
+	name      string
+	namespace string
+	group     string
+	version   string
+	resource  string
+}
+
+var (
+	// Update this array to contain objects which should have been deleted in this release. For example:
+	//	object{kind: "Namespace",
+	//		namespace: "",
+	//		name:      "openshift-foobar"},
+	//
+	deletes = []object{}
+
+	// Update this array to contain unstructured objects which should have been deleted in this release. For example:
+	//	unstructured{name: "dns",
+	//		namespace: "openshift-dns-operator",
+	//		group:     "monitoring.coreos.com",
+	//		version:   "v1",
+	//		resource:  "prometheusrules"},
+	//
+	unstructuredDeletes = []unstructured{}
```

Comment:
can we fill some of these in now?  Doesn't block this PR, but if we have any ready, populating them now saves us a follow-up PR later.

Reply:
yes we can